packages/zowe-explorer/README.md

```diff
@@ -70,7 +66,7 @@ When environment conditions do not support the Zowe CLI built-in Credential Mana
 
 ### Multifactor authentication support (MFA)
 
-Zowe Explorer supports the use of MFA tokens for authentication. When using zOSMF profiles it is required to connect through the Zowe API Mediation Layer (API ML). Suggested use case for MFA authentication using Zowe Explorer is by logging into the API ML via the right-click login action in Zowe Explorer and when prompted for password enter the MFA token in place of user password, the API will return a JSON token (JWT) that will be used for further authentication.
+Zowe Explorer supports the use of MFA tokens for authentication. To use MFA when using zOSMF profiles, it is required to connect through the Zowe API Mediation Layer (API ML). The suggested use case for MFA authentication using Zowe Explorer is to log into the API ML via the right-click login action in Zowe Explorer and, when prompted for password, enter the MFA token in place of the password. Then the API will return a JSON token (JWT) that will be used for further authentication.
```

Comment:
Try to avoid future tense: " ... Then the API returns a JSON token (jwt) that is used for further authentication."